packages/make-angular-cli-faster/src/utilities/make-angular-cli-faster.ts

```diff
@@ -42,4 +43,27 @@ export async function makeAngularCliFaster(args: Args) {
       'and how it can speed up your CI by up to 10 times at https://nx.dev/getting-started/nx-and-angular',
     ],
   });
+
+  if (migration.incompatibleWithAngularVersion) {
+    output.warn({
+      title:
+        'The provided version of Nx is not compatible with the currently installed version of Angular.',
+      bodyLines: [
+        'The workspace was still migrated, but you might find unexpected issues or behaviors using this version.',
+        'To know which versions are compatible, please refer to https://nx.dev/angular-nx-version-matrix.',
+      ],
+    });
+  }
+
+  // if migrated to an older version than root project is supported, log a warning
+  if (lt(migration.version, '15.3.0-beta.5')) {
+    output.warn({
+      title: 'Your workspace was not migrated to the latest version of Nx.',
+      bodyLines: [
+        `You might find some differences compared to the documentation which reflects the latest state of Nx.`,
+        'You could consider update your workspace one major version at a time to the latest version.',
```

Comment:
This is basically telling them to run `nx migrate ${majorVersion + 1}` right?

Reply:
Exactly. That's what we recommend when migrating Nx, one major version at a time.